content_filter.py

```
@@ -1585,7 +1624,7 @@ async def log_censored_display_name(member, censored_display_name, server_config
     embed.add_field(name="User Link", value=f"[Go to User](https://discord.com/users/{member.id})", inline=False)
     await log_channel.send(embed=embed)
 
-async def log_scan_deletion(message: discord.Message, censored_message: str, server_config: dict):
+async def log_scan_deletion(message: discord.Message, censored_message: str, server_config: dict, removed_attachments: List[discord.Attachment] = [], blocked_exts: set = set()):
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)